test/functional/core/job_spec.lua

```diff
@@ -115,6 +115,19 @@ describe('jobs', function()
     ok(string.find(err, "E475: Invalid argument: expected valid directory$") ~= nil)
   end)
 
+  it('produces error when using non-executable `cwd`', function()
+    if iswin() then return end  -- N/A for Windows
```

Comment:
Did the test fail on appveyor? 

The Vim patch doesn't seem to skip Windows, although the problem in Nvim could be related to https://github.com/neovim/neovim/issues/8244

Reply:
I assumed that calling `chmod` would cause the test to fail and that making a directory executable on Windows wasn't really possible.  Even if the test would pass, I figured it should be skipped on Windows because the description wouldn't make sense anyways.